dbt/models/default/default.vw_pin_universe_temp.sql

```diff
@@ -0,0 +1,3 @@
+-- Small, managable version of pin universe for the purporse of updating the
+-- Parcel Universe asset on open data.
+SELECT * FROM {{ ref('location.vw_pin10_location') }} LIMIT 100000
```

Comment:
**[Question, non-blocking]** Does it matter that this view won't have the same schema as `default.vw_pin_universe`?

Reply:
Yes, thank you for catching this Jean. I completely got this wrong.